### PR TITLE
Add SoftCreatR Media

### PR DESCRIPTION
**Pull request by SoftCreatR**
n/t

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -6808,6 +6808,19 @@
...
+    {
+        "name": "SoftCreatR Media",
+        "url": "https://www.softcreatr.com/account-management/",
+        "difficulty": "easy",
+        "notes": "Your account will be deleted after 7 days, if you don't cancel the deletion process before. Created content (such as forum posts) won't be deleted, but your username will be anonymized and protected from re-registration for 363 days.",
+        "notes_de": "Dein Benutzerkonto wird nach Ablauf von 7 Tagen gelöscht, solltest du den Vorgang vorher nicht abbrechen. Erstellte Inhalte (wie etwa Foren-Beiträge) werden nicht gelöscht, jedoch wird dein Benutzername anonymisiert und für 363 Tage vor einer erneuten Registrierung geschützt.",
+        "domains": [
+            "www.softcreatr.com",
+            "support.softcreatr.com",
+            "shop.softcreatr.com",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
            "shop.softcreatr.com"
```
Remove the comma to be a proper JSON

File: _data/sites.json [hunk 2]
```diff
@@ -6808,6 +6808,19 @@
             "socialblade.com"
         ]
     },
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```

File: _data/sites.json [hunk 3]
```diff
@@ -6809,6 +6809,19 @@
...
     },
 
+    {
+        "name": "SoftCreatR Media",
+        "url": "https://www.softcreatr.com/account-management/",
+        "difficulty": "easy",
+        "notes": "Your account will be deleted after 7 days, if you don't cancel the deletion process before. Created content (such as forum posts) won't be deleted, but your username will be anonymized and protected from re-registration for 363 days.",
+        "notes_de": "Dein Benutzerkonto wird nach Ablauf von 7 Tagen gelöscht, solltest du den Vorgang vorher nicht abbrechen. Erstellte Inhalte (wie etwa Foren-Beiträge) werden nicht gelöscht, jedoch wird dein Benutzername anonymisiert und für 363 Tage vor einer erneuten Registrierung geschützt.",
+        "domains": [
+            "www.softcreatr.com",
```
Comment (tupaschoal) [inline _data/sites.json]: Just one domain `softcreatr.com` is enough, no need for the `www` and the other two with `support.` and `shop.`

Comment (SoftCreatR) [reply _data/sites.json]: All good things are 3 ;)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**